<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddProduct</class>
 <widget class="QWidget" name="AddProduct">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>751</width>
    <height>670</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QScrollArea" name="scrollArea_200">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>717</width>
        <height>670</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <spacer name="horizontalSpacer_203">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>186</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_209">
         <item>
          <spacer name="verticalSpacer_213">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_200">
           <property name="font">
            <font>
             <family>Calibri</family>
             <pointsize>14</pointsize>
             <bold>false</bold>
            </font>
           </property>
           <property name="title">
            <string>Submit product</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_8">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_207">
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_203">
                <item>
                 <spacer name="verticalSpacer_200">
                  <property name="orientation">
                   <enum>Qt::Vertical</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>20</width>
                    <height>40</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <widget class="QLabel" name="label_200">
                  <property name="font">
                   <font>
                    <family>Calibri</family>
                    <pointsize>14</pointsize>
                    <bold>false</bold>
                   </font>
                  </property>
                  <property name="text">
                   <string>Product name</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="lineEdit_203">
                  <property name="maxLength">
                   <number>100</number>
                  </property>
                  <property name="placeholderText">
                   <string>Insert name</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="label_202">
                  <property name="font">
                   <font>
                    <family>Calibri</family>
                    <pointsize>9</pointsize>
                    <bold>false</bold>
                   </font>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">color: rgb(255, 0, 0);</string>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_202">
                <item>
                 <spacer name="verticalSpacer_203">
                  <property name="orientation">
                   <enum>Qt::Vertical</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>20</width>
                    <height>40</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <widget class="QLabel" name="label_203">
                  <property name="font">
                   <font>
                    <family>Calibri</family>
                    <pointsize>14</pointsize>
                    <bold>false</bold>
                   </font>
                  </property>
                  <property name="text">
                   <string>Product description</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="lineEdit_202">
                  <property name="placeholderText">
                   <string>Write your description</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <widget class="QLabel" name="label_204">
                <property name="font">
                 <font>
                  <family>Calibri</family>
                  <pointsize>9</pointsize>
                  <bold>false</bold>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true">color: rgb(255, 0, 0);</string>
                </property>
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_200">
                <item>
                 <spacer name="verticalSpacer_205">
                  <property name="orientation">
                   <enum>Qt::Vertical</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>20</width>
                    <height>40</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <widget class="QLabel" name="label_205">
                  <property name="font">
                   <font>
                    <family>Calibri</family>
                    <pointsize>14</pointsize>
                    <bold>false</bold>
                   </font>
                  </property>
                  <property name="text">
                   <string>Price</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="lineEdit_204">
                  <property name="maxLength">
                   <number>8</number>
                  </property>
                  <property name="placeholderText">
                   <string>$</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="label_206">
                  <property name="font">
                   <font>
                    <family>Calibri</family>
                    <pointsize>9</pointsize>
                    <bold>false</bold>
                   </font>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">color: rgb(255, 0, 0);</string>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_204">
                <item>
                 <spacer name="verticalSpacer_207">
                  <property name="orientation">
                   <enum>Qt::Vertical</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>20</width>
                    <height>40</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <widget class="QLabel" name="label_207">
                  <property name="font">
                   <font>
                    <family>Calibri</family>
                    <pointsize>14</pointsize>
                    <bold>false</bold>
                   </font>
                  </property>
                  <property name="text">
                   <string>Product categories</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QComboBox" name="comboBox">
                  <property name="currentIndex">
                   <number>-1</number>
                  </property>
                  <property name="placeholderText">
                   <string>Choose a category</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="label_209">
                  <property name="font">
                   <font>
                    <family>Calibri</family>
                    <pointsize>9</pointsize>
                    <bold>false</bold>
                   </font>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">color: rgb(255, 0, 0);</string>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="verticalSpacer_208">
                  <property name="orientation">
                   <enum>Qt::Vertical</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>20</width>
                    <height>40</height>
                   </size>
                  </property>
                 </spacer>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_205">
                <item>
                 <widget class="QLabel" name="label_208">
                  <property name="font">
                   <font>
                    <family>Calibri</family>
                    <pointsize>14</pointsize>
                    <bold>false</bold>
                   </font>
                  </property>
                  <property name="text">
                   <string>Quantity</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QSpinBox" name="spinBox_200">
                  <property name="minimum">
                   <number>1</number>
                  </property>
                  <property name="maximum">
                   <number>5000</number>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="verticalSpacer_210">
                  <property name="orientation">
                   <enum>Qt::Vertical</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>20</width>
                    <height>40</height>
                   </size>
                  </property>
                 </spacer>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_206">
                <item>
                 <widget class="QLabel" name="label_210">
                  <property name="font">
                   <font>
                    <family>Calibri</family>
                    <pointsize>14</pointsize>
                    <bold>false</bold>
                   </font>
                  </property>
                  <property name="text">
                   <string>Product image</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout">
                  <item>
                   <widget class="QPushButton" name="pushButton_250">
                    <property name="text">
                     <string>Choose File...</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLabel" name="label_250">
                    <property name="text">
                     <string/>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <spacer name="horizontalSpacer">
                    <property name="orientation">
                     <enum>Qt::Horizontal</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>40</width>
                      <height>20</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                 </layout>
                </item>
                <item>
                 <widget class="QLabel" name="label_212">
                  <property name="font">
                   <font>
                    <family>Calibri</family>
                    <pointsize>9</pointsize>
                    <bold>false</bold>
                   </font>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">color: rgb(255, 0, 0);</string>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="verticalSpacer_212">
                  <property name="orientation">
                   <enum>Qt::Vertical</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>20</width>
                    <height>40</height>
                   </size>
                  </property>
                 </spacer>
                </item>
               </layout>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_200">
              <item>
               <spacer name="horizontalSpacer_200">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton_200">
                <property name="text">
                 <string>Submit</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_202">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_214">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="horizontalSpacer_204">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>185</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
